payments_service/solitude/__init__.py

```diff
@@ -37,15 +37,36 @@ class SolitudeBodyguard(APIView):
     # resource = 'services.status'
     resource = None
 
-    def _api_request(self, method, *args, **kw):
+    def replace_call_args(self, django_request, args, kw):
```

Comment:
Maybe a note about `django_request`, actually surprised flake8 doesn't complain about it be defined but never used or returns. Oh man if only we had interfaces.